sidebars.js

```diff
@@ -1,72 +1,87 @@
-/**
- * Creating a sidebar enables you to:
- - create an ordered group of docs
- - render a sidebar for each doc of that group
- - provide next/previous navigation
-
- The sidebars can be generated from the filesystem, or explicitly defined here.
-
- Create as many sidebars as you want.
- */
-
 // @ts-check
 
 /** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
 const sidebars = {
-  // By default, Docusaurus generates a sidebar from the docs folder structure
-  docSidebar: [
-    { type: "autogenerated", dirName: "." },
-    {
-      type: "category",
-      label: "SDKs",
-      items: [
-        {
-          type: "link",
-          label: "Rust Client",
-          href: "https://github.com/reductstore/reduct-rs",
-        },
+    docSidebar: [
+        "getting-started",
+        "configuration",
+        "how-does-it-work",
         {
-          type: "link",
-          label: "Python Client",
-          href: "https://github.com/reductstore/reduct-py",
+            type: "category",
+            label: "Guides",
+
+            collapsed: false,
+            collapsible: false,
+            items: [
+                "guides/buckets",
+                "guides/data-ingestion",
+                "guides/data-querying",
+                "guides/data-replication",
+                {
+                    type: "category",
+                    label: "Integration",
+                    items: [
+                        "guides/integration/azure",
```

Comment:
I think it would more intuitive to have the integration tab at the doc level
- Getting started
- ..
- Guides
  - ... 
- Integration
  - Azure
  - e.g. EMQX
  - ..
- ...